be/src/common/elf.cpp

```diff
@@ -40,21 +41,22 @@ Elf::Elf(const std::string& path) {
     std::error_code ec;
     elf_size = std::filesystem::file_size(_file, ec);
     if (ec) {
-        LOG(FATAL) << fmt::format("failed to get file size {}: ({}), {}", _file.native(),
-                                  ec.value(), ec.message());
+        throw Exception(Status::IOError("failed to get file size {}: ({}), {}", _file.native(),
```

Comment:
please add a new error type. we can call it `FatalError`. and for all Exception changed from `LOG(FATAL)`, use it as the error code